index/templates/newindex.html

```diff
@@ -0,0 +1,109 @@
+<!--
```

Comment:
Sorry that I forgot to change the filename,and I just changed the filename to index.html